src/client/components/map/RectangleSelectionTool.ts

```diff
@@ -47,7 +47,9 @@ const RectangleSelectionTool: ISelectionTool = {
     staticMetadata: IStaticMetadata,
     districtsDefinition: DistrictsDefinition,
     lockedDistricts: LockedDistricts,
-    staticGeoLevels: UintArrays
+    staticGeoLevels: UintArrays,
+    // eslint-disable-next-line
+    setActive: (isActive: boolean) => {}
```

Comment:
We want to [return void](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#return-types-of-callbacks) here:
```suggestion
    setActive: (isActive: boolean) => void
```